aspnetcore/signalr/api-design.md

```diff
@@ -75,3 +75,6 @@ The older clients won't be expecting the `Sender` value, so they'll ignore it. A
 [!code-typescript[OnWithObjectNew](api-design/sample/Samples.ts?name=OnWithObjectNew&highlight=2-5)]
 
 In this case, the new client is also tolerant of an old server that doesn't provide the `Sender` value. Since the old server won't provide the `Sender` value, the client checks to see if it exists before accessing it.
+
+> [!NOTE]
+> ASP.NET Core SignalR no longer depends on getting many assemblies from NuGet, those assemblies are implicitly referenced in the `Microsoft.NET.Sdk.Web`. For more information, see [Differences between ASP.NET SignalR and ASP.NET Core SignalR](xref:signalr/version-differences#how-to-identify-the-signalr-version).
```

Comment:
This needs to be an include

Reply:
OK, I will move it into an include, thanks.